expression/builtin_time.go

```diff
@@ -1770,6 +1770,60 @@ func getTimeZone(ctx context.Context) *time.Location {
 	return ret
 }
 
+//isDuration returns a boolean indicating whether the str is duration
+func isDuration(str string) bool {
```

Comment:
Add test cases for this function.

I think a regular expression will be more explicit.

what will `isDuration("aa:bb:cc")` return?